commands/azureCommands/acr-build-logs-utils/logScripts.js

```diff
@@ -0,0 +1,152 @@
+// Global Variables
+const status = {
+    'Succeeded': 4,
+    'Queued': 3,
+    'Error': 2,
+    'Failed': 1
+}
+
+var currentN = 4;
+var currentDir = "asc"
+
+// Main
+let content = document.querySelector('#core');
+const vscode = acquireVsCodeApi();
+setLoadMoreListener();
+setTableSorter();
+
+/* Sorting
+ * PR note, while this does not use a particularly quick algorithm
+ * it allows a low stuttering experience that allowed rapid testing.
+ * I will improve it soon.*/
+function sortTable(n, dir = "asc", holdDir = false) {
+    currentDir = dir;
+    currentN = n;
+    let table, rows, switching, i, x, y, shouldSwitch, switchcount = 0;
+    let cmpFunc = acquireCompareFunction(n);
+    table = document.getElementById("core");
+    switching = true;
+    //Set the sorting direction to ascending:
+
+    while (switching) {
+        switching = false;
+        rows = table.getElementsByClassName("holder");
+        for (i = 0; i < rows.length - 1; i++) {
```

Comment:
Did ts lint have a problem with this for loop?

Reply:
It doesn't, this loop is bounded differently, note the length -1